File: JavaMessageNX/Consumer.java
import java.util.Map;

// javac -cp ".;libs\jars\*" .\Consumer.java
//  java -cp ".;libs\jars\*" Consumer
public class Consumer {

  public static void main(String[] args) throws Exception {
    MessageNX mnx = new MessageNX();
    mnx.setChannel("appId");
    mnx.consume(
      new MessageNX.process() {
        @Override
        public void processor(Map<String, Object> topic) {
          System.out.println(topic.get("name"));
        }
      }
    );
  }
}

File: C#MessageNX/Consumer.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;

class Consumer
{
    static async Task Main(string[] args)
    {
        var mnx = new MessageNX();
        mnx.SetChannel("141983614");
        await mnx.ConsumeAsync(topic =>
        {
            if (topic.ContainsKey("name"))
            {
                Console.WriteLine(topic["name"]);
            }
        });
    }
}